app/web/src/api/sdf/dal/component.ts

```diff
@@ -57,7 +57,7 @@ export interface RawComponent {
   resourceId: string;
   id: ComponentId;
   componentType: ComponentType;
-  parentId?: ComponentId;
+  parentId?: undefined;
```

Comment:
Why not just remove the property?

Reply:
I started with that; but I didn't want to have to surgically edit the code that uses the properties (it's all unused code, but it's a separate project to dump it all). Honestly didn't even need to do this, but it felt a little more honest.